scraper/src/fetch_german_names.js

```diff
@@ -0,0 +1,212 @@
+import axios from "axios";
+import axiosRetry from "axios-retry";
+import fs from "fs";
+import { parse as json2csv } from "json2csv";
+import csv from "csvtojson";
+import { capitalizeWords } from "./helpers/helpers.js";
+import { log } from "console";
+
+let GermanNamesFound = 0;
+
+/**
+ * Defines the amount of retries we do, if axios encounters errors during a HTTP GET Request.
+ * Increse Delay if we encounter error to prevent 429 Errors.
```

Comment:
```suggestion
 * Increase delay if we encounter error to prevent 429 Errors.
```

Reply:
Thank you for the review!
I implemented your requests